__tests__/app.test.js

```diff
@@ -66,15 +72,49 @@ describe("Get", () => {
 				});
 		});
 	});
+	describe("api/reviews/:review_id", () => {
+		test("200-responses with a single object", () => {
+			return request(app)
+				.get("/api/reviews/1")
+				.expect(200)
+				.then(({ body }) => {
```

Comment:
Here we know which review we are getting back, so we could assert against actual values